HW3/AmountOfMoneySpent.java

```diff
@@ -0,0 +1,38 @@
+package HW3;
+
+import java.util.Scanner;
+
+public class AmountOfMoneySpent {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Are you VIP client?  true/false");
+        boolean isVipClient = scanner.nextBoolean();
+
+        System.out.println("Enter your age");
+        int minage = scanner.nextInt();
+
+        int entryAmount = 5000;
+        int orderAmount;
+        int totalAmountSpent;
+
+        if (minage >= 16) {
+            if (isVipClient == false) {
+                System.out.println("Enter your order amount");
+                orderAmount = scanner.nextInt();
```

Comment:
You write these 2 lines - input `orderAmount` and message anyway and it doesn't depend on any condition, no need to duplicate and specify them here and in `else` clause.

Reply:
Please confirm last changes in this file. Thanks!